.eslintignore

```diff
@@ -8,6 +8,7 @@ dist.3p
 dist.tools
 out
 examples
+firebase
```

Comment:
Curious: When is this directory generated?

Reply:
when you run `gulp firebase` to upload a build to a firebase project
https://github.com/ampproject/amphtml/blob/df859fa69953b7f51482d339f8485a88ef6403a5/contributing/TESTING.md#L113-L116